selfdrive/car/volkswagen/interface.py

```diff
@@ -230,7 +230,7 @@ def _update(self, c):
       events.add(EventName.belowSteerSpeed)
 
     if self.CS.CP.openpilotLongitudinalControl:
-      if ret.vEgo < self.CP.minEnableSpeed + 2.:
+      if ret.vEgo < self.CP.minEnableSpeed + 0.5:
```

Comment:
I was 50/50 on it, had originally thought to give the minSpeed folks another ~4.5mph of room to operate, but on further thought I think it's probably better to have the additional notification "space" on slowdown before forced disengage.

Reply:
It's only a noEntry, it wouldn't force disengage would it?

Reply:
Oh, I thought it threw a silent alert like preLaneChange or a DM alert at standstill, but see that I was wrong. In that case, I don't think there's great benefit to the proposed change, what was there is sufficient to prevent faults. Will revert.